migration-util/src/main/java/edu/unc/lib/dcr/migration/MigrationCLI.java

```diff
@@ -85,6 +85,11 @@
             description = "Port for redis. Default 6379.")
     protected int redisPort;
 
+    @Option(names = {"--display-label"},
```

Comment:
since this option only applies to SubmitDepositCommand and TransformContentCommand, could you add the option directly to those to keep down the clutter in the --help option?